features/device-config/retrieval/src/main/java/org/opennms/features/deviceconfig/retrieval/impl/RetrieverImpl.java

```diff
@@ -221,9 +223,14 @@ static String timeoutFailureMsg(String host, int port) {
         return "device config was not received in time. host: " + host + "; port: " + port;
     }
 
-    private static String uniqueFilenameSuffix() {
-        return "monitor" + String.valueOf(uploadCounter.incrementAndGet());
+    private static byte[] uuidToBytes(UUID uuid) {
+        ByteBuffer bb = ByteBuffer.wrap(new byte[16]);
+        bb.putLong(uuid.getMostSignificantBits());
+        bb.putLong(uuid.getLeastSignificantBits());
+        return bb.array();
     }
 
-    private static AtomicLong uploadCounter = new AtomicLong();
+    private static String uniqueFilenameSuffix() {
```

Comment:
Add a comment to make sure `monitor` will not be removed in future.

Reply:
done.